### PR TITLE
Export data to a file

### PR DESCRIPTION
**Pull request by omri312**
New PR to develop

### CODE REVIEW

**Comment by omri312** [comment]
It seems that this was implemented in the manager, but should now move to the supervisor, then this can be merged

File: src/corona_hakab_model/manager.py [hunk 1]
```diff
@@ -131,7 +131,26 @@ def run(self):
...
+
+    def Export(self):
+        consts_attr = [attr for attr in dir(self.consts) if not callable(getattr(self.consts, attr)) and not attr.startswith("_")]
+        output_dict = {}
+        for attr in consts_attr:
+            output_dict[attr] = getattr(self.consts,attr)
+        for attr in self.per_generation:
+            output_dict[str(attr)] = self.per_generation[attr]
+        
+        filehandle = open("..\output\Export.json","w")
```
Comment (mor-m-shemesh) [inline src/corona_hakab_model/manager.py:144]: Why have it be a static file path? just get it as an argument

File: src/corona_hakab_model/manager.py [hunk 2]
```diff
@@ -131,7 +131,26 @@ def run(self):
...
+        export_string = "{\n"
+        for key, val in output_dict.items():
+            export_string = export_string + '"' + str(key) + '": ' + str(val) + ',\n'
```
Comment (mor-m-shemesh) [inline src/corona_hakab_model/manager.py:147]: Why not use json.dump?
It seems you are already exporting only simple data types

File: src/corona_hakab_model/manager.py (hunk 1 above)
Comment (mor-m-shemesh) [reply src/corona_hakab_model/manager.py:144]: Also, always use 
with open(<file path>, 'wb') as f:
    ....
so you don't mess with handles to files.

File: src/corona_hakab_model/manager.py (hunk 2 above)
Comment (mishana) [inline src/corona_hakab_model/manager.py:147]: an f-string will look much better here

**Comment by omri312** [comment]
I believe this is irrelevant, closing.

**Pull request closed without merging**